ovirt_metrics.gemspec

```diff
@@ -19,9 +19,9 @@ Gem::Specification.new do |spec|
   spec.test_files   += %w[.rspec]
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.1"
+  spec.required_ruby_version = ">= 2.3.1"
 
-  spec.add_development_dependency "bundler", "~> 1.3"
```

Comment:
I think you can remove this dependency.

Reply:
I think we won't be able to build with a dependency on v2 because the appliance is built with v1.15.4 IIRC

Reply:
Ah